### ISSUE TITLE
Update to lastest version(2.0.519) an excetpion occurred:Failed to load file or assembly System.Runtime.InteropServices.RuntimeInformation

### ISSUE THREAD

**Issue by hejieyuan2005** [StackExchange/StackExchange.Redis#1018]
Hello guys:

I have a test program that supports running , an exception has occurred after my update to the latest version ...

My OS version: windows7 x64

Program Running Environment Information before Version Update：
     StackExchange.Redis(1.2.6)
     VS2013 & NetFramework 4.5.1

Program Running Environment Information after Version Update : 
     StackExchange.Redis(2.0.519)
     VS2013 & NetFramework 4.6.1

**System.AggregateException
  HResult=-2146233088
  Message=One or more errors occurred。
  Source=mscorlib
  StackTrace:
       in System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
       in System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
       in System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout)
       in StackExchange.Redis.ConnectionMultiplexer.ConnectImpl(Object configuration, TextWriter log)
       in StackExchange.Redis.ConnectionMultiplexer.Connect(ConfigurationOptions configuration, TextWriter log)
       in Common.Redis..ctor() at e:\Net\redis-client\StackExchange.Redis.demo\RedisTest\Common\Redis.cs:line number 38
       in RedisPro.Program.Main(String[] args) at e:\Net\redis-client\StackExchange.Redis.demo\RedisTest\RedisPro\Program.cs:line number 14
       in System.AppDomain._nExecuteAssembly(RuntimeAssembly assembly, String[] args)
       in System.AppDomain.ExecuteAssembly(String assemblyFile, Evidence assemblySecurity, String[] args)
       in Microsoft.VisualStudio.HostingProcess.HostProc.RunUsersAssembly()
       in System.Threading.ThreadHelper.ThreadStart_Context(Object state)
       in System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       in System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       in System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
       in System.Threading.ThreadHelper.ThreadStart()
  InnerException: System.TypeInitializationException
       HResult=-2146233036
       Message=“StackExchange.Redis.PerfCounterHelper” type initializer throws an exception。
       Source=StackExchange.Redis
       TypeName=StackExchange.Redis.PerfCounterHelper
       StackTrace:
            in StackExchange.Redis.PerfCounterHelper.GetThreadPoolAndCPUSummary(Boolean includePerformanceCounters)
            in StackExchange.Redis.ResultProcessor.SetResult(PhysicalConnection connection, Message message, RawResult result)
       InnerException: System.IO.FileNotFoundException
            HResult=-2147024894
            Message=Failed to load file or assembly “System.Runtime.InteropServices.RuntimeInformation, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a” or one of its dependencies, system could not find the file。
            Source=StackExchange.Redis
            FileName=System.Runtime.InteropServices.RuntimeInformation, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
            FusionLog==== Prebound status information ===
Log: DisplayName = System.Runtime.InteropServices.RuntimeInformation, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 (Fully-specified)
Log: Appbase = file:///E:/Net/redis-client/StackExchange.Redis.demo/RedisTest/RedisPro/bin/Debug/
Log: initial PrivatePath = NULL
Call assemblies: StackExchange.Redis, Version=2.0.0.0, Culture=neutral, PublicKeyToken=c219ff1ca8c2ce46。
===**

**Comment by NickCraver** [comment]
This was a `netstandard2.0`-only build problem. Updating to later versions resolves it was due to issues _just like this_ we added back `net462` and `net472` builds to reduce (but not eliminate) assembly loading issues.

TL;DR: If hitting this, grab latest 2.x please!
